speech/cloud-client/src/main/java/com/example/speech/InfiniteStreamRecognize.java

```diff
@@ -253,13 +270,12 @@ public void onError(Throwable t) {}
                   bridgingOffset = finalRequestEndTime;
                 }
                 int chunksFromMS = (int) Math.floor((finalRequestEndTime
-                                                - bridgingOffset) / chunkTime);
```

Comment:
Can these whitespace changes all be removed from the diff?

A follow-up can be sent with reformatting _if reformatting is required_.